datafusion/expr/src/expr.rs

```diff
@@ -1843,7 +1623,7 @@ impl Expr {
     /// ```
     /// # use datafusion_expr::col;
     /// # use std::collections::HashMap;
-    /// # use datafusion_expr::expr::FieldMetadata;
+    /// # use datafusion_common::metadata::FieldMetadata;
```

Comment:
one way we have tried to help users on upgrade is to leave a `pub use` when we move the struct

In this case, I think thatw means if you put this line in `datafusion_expr::expr`
```rust
pub use datafusion_common::metadata::FieldMetadata;
```

Existing code will still compile